packages/internal/src/build/babel/web.ts

```diff
@@ -1,40 +1,186 @@
 import fs from 'fs'
 import path from 'path'
 
+import type { TransformOptions } from '@babel/core'
+
 import { getPaths } from '../../paths'
 
-import { registerBabel, RegisterHookOptions } from './common'
+import {
+  CORE_JS_VERSION,
+  getCommonPlugins,
+  registerBabel,
+  RegisterHookOptions,
+} from './common'
+
+export const getWebSideBabelPlugins = () => {
+  const rwjsPaths = getPaths()
+
+  const plugins: TransformOptions['plugins'] = [
+    ...getCommonPlugins(),
+
+    // === Import path handling
+    [
+      require('../babelPlugins/babel-plugin-redwood-src-alias').default,
+      {
+        srcAbsPath: rwjsPaths.web.src,
+      },
+      'rwjs-babel-src-alias',
+    ],
+    [
+      require('../babelPlugins/babel-plugin-redwood-directory-named-import')
+        .default,
+      undefined,
+      'rwjs-directory-named-modules',
+    ],
 
-// TODO: move web side babel plugins here too when we pretranspile web side
-// and export getWebSideBabelPlugins
+    // === Auto imports, and transforms
+    [
+      'babel-plugin-auto-import',
+      {
+        declarations: [
+          {
+            // import { React } from 'react'
+            default: 'React',
+            path: 'react',
+          },
+          {
+            // import PropTypes from 'prop-types'
+            default: 'PropTypes',
+            path: 'prop-types',
+          },
+          {
+            // import gql from 'graphql-tag'
+            default: 'gql',
+            path: 'graphql-tag',
+          },
+        ],
+      },
+      'rwjs-web-auto-import',
+    ],
+    ['babel-plugin-graphql-tag', undefined, 'rwjs-babel-graphql-tag'],
+    [
+      'inline-react-svg',
+      {
+        svgo: {
+          plugins: [
+            {
+              name: 'removeAttrs',
+              params: { attrs: '(data-name)' },
+            },
+            // Otherwise having style="xxx" breaks
+            'convertStyleToAttrs',
+          ],
+        },
+      },
+      'rwjs-inline-svg',
+    ],
+
+    // === Handling redwood "magic"
+  ].filter(Boolean)
+
+  return plugins
+}
+
+export const getWebSideOverrides = (
+  { staticImports } = {
+    staticImports: false,
+  }
+) => {
+  const overrides = [
+    {
+      test: /.+Cell.(js|tsx)$/,
```

Comment:
The two different styles here between "Routes" and "Cells" - maybe we should stick to a single one.

Reply:
👌🏾 I just copied it over haha

Reply:
✅